modelling/src/main/java/org/axonframework/modelling/saga/SagaLifecycle.java

```diff
@@ -19,6 +19,8 @@
 import org.axonframework.messaging.Scope;
```

Comment:
Nit: Whenever you touch a file, the copyright notice should update to the year we're in.